### ISSUE TITLE
agent return a strange state

### ISSUE THREAD

**Issue by onlytailei** [Microsoft/malmo-challenge#23]
python pig_chase_dqn.py 
```
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "pig_chase_dqn.py", line 122, in agent_factory
    action = agent.act(obs, reward, agent_done, is_training=True)
  File "/home/tai/Freiburg/Malmo/malmo-challenge/ai_challenge/pig_chase/agent.py", line 73, in act
    return self.current_agent.act(new_state, reward, done, is_training)
  File "/home/tai/Freiburg/Malmo/malmo-challenge/ai_challenge/pig_chase/agent.py", line 107, in act
    me = [(j, i) for i, v in enumerate(state) for j, k in enumerate(v) if self.name in k]
TypeError: 'numpy.uint8' object is not iterabl
```

It is not a (9,9) matrix but a (84,1) list and all the elements are int.

**Comment by mfuntowicz** [comment]
Hi @onlytailei 

Thank for reporting this, and sorry for the inconvenience. 

I did something wrong while merging the dqn_model branch. I'm working on this, and hope be able to fix it by today or tomorrow.

Thanks, 
Morgan

**Comment by mfuntowicz** [comment]
Should be fixed in #25
